### PR TITLE
HDDS-9587. `No serializer found for class ThreadLocal` in DBSCanner for pipelines in scm.db

### PR DESCRIPTION
**Pull request by mladjan-gadzic**
## What changes were proposed in this pull request?
Fix for an exception which occurs during serialization of nodesInOrder when using ldb scan for scm.db and pipelines column family.

## What is the link to the Apache JIRA
[https://issues.apache.org/jira/browse/HDDS-9587](https://issues.apache.org/jira/browse/HDDS-9587)

## How was this patch tested?
Manual tests

### CODE REVIEW

**Review by xBis7 — APPROVED** [review]
Thanks @mladjan-gadzic. Changes LGTM!

**Comment by adoroszlai** [comment]
Thanks @mladjan-gadzic for the patch.  Can you please add a test case in `TestLDBCli`?

**Comment by mladjan-gadzic** [comment]
@xBis7, @adoroszlai thanks for the review. Test is added.

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/debug/TestLDBCli.java [hunk 1]
```diff
@@ -236,6 +238,30 @@ void testLDBScan(
...
+    List<String> completeScanArgs = new ArrayList<>(Arrays.asList(
+        "--db", dbStore.getDbLocation().getAbsolutePath(),
+        "scan",
+        "--column-family", PIPELINES));
+
+    int exitCode = cmd.execute(completeScanArgs.toArray(new String[0]));
+    // Check exit code. Print stderr if not expected
+    Assertions.assertEquals(0, exitCode, stderr.toString());
+
+    if (exitCode == 0) {
```
Comment (adoroszlai) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/debug/TestLDBCli.java:256]: Nit: do we need this `if` after asserting the same?

**Review by adoroszlai — COMMENTED** [review]
Thanks @mladjan-gadzic for adding the test.

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/debug/TestLDBCli.java (hunk 1 above)
Comment (mladjan-gadzic) [reply hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/debug/TestLDBCli.java]: No need, it is a redundant. Thanks for noticing.

**Comment by adoroszlai** [comment]
Thanks @mladjan-gadzic for the patch, @xBis7 for the review.

**Pull request merged**